CHANGELOG.md

```diff
@@ -5,9 +5,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [7.0.0]
+### Changed
+- **BREAKING:** Bump minimum Node version to 16 ([#139](https://github.com/MetaMask/json-rpc-engine/pull/139))
+- **BREAKING:** Use `@metamask/utils` ([#105](https://github.com/MetaMask/json-rpc-engine/pull/105))
```

Comment:
Hmm, it's unclear how this is breaking. We should include some guidance here if we can on how to accommodate any breaking changes made. Reading up on this PR now.

Reply:
I see; I was wondering if there were any functional changes made here beyond that.

It looks like that PR also replaced a bunch of internal errors with our standard RPC errors. But presumably the old errors would have been wrapped in an internal RPC error, so maybe the impact there is limited.